octobot_trading/personal_data/positions/position.py

```diff
@@ -734,16 +734,16 @@ def to_dict(self):
             enums.ExchangeConstantsPositionColumns.STATUS.value: self.status.value,
             enums.ExchangeConstantsPositionColumns.TIMESTAMP.value: self.timestamp,
             enums.ExchangeConstantsPositionColumns.SIDE.value: self.side.value,
-            enums.ExchangeConstantsPositionColumns.QUANTITY.value: self.quantity,
-            enums.ExchangeConstantsPositionColumns.SIZE.value: self.size,
-            enums.ExchangeConstantsPositionColumns.NOTIONAL.value: self.value,
+            enums.ExchangeConstantsPositionColumns.CONTRACT_SIZE.value: self.quantity,
+            enums.ExchangeConstantsPositionColumnsPOSITION_SIZE.value: self.size,
+            enums.ExchangeConstantsPositionColumns.POSITION_VALUE.value: self.value,
```

Comment:
we will also have to ensure compatibiliy on those values for things using them

Reply:
I think I got all of them now (but better check as well), in octobot services is also a commit